index.html

```diff
@@ -3,6 +3,7 @@
     <head>
         <meta charset="UTF-8" />
         <title>Powerhouse Connect</title>
+        <base href="/">
```

Comment:
This allows using `BrowserRouter` instead of `HashRouter`.
There was an issue loading `icons.svg` from the design system before. It would try to load from the current path `/d/core-dev/icons.svg`